scripts/finish-release.test.ts

```diff
@@ -0,0 +1,52 @@
+import * as finishRelease from './finish-release';
```

Comment:
should we have `scripts/__tests__/` dir?

Reply:
Added